Reflective Report.tex

```diff
@@ -0,0 +1,94 @@
+\author{1503048}
+
+\begin{document}
+
+\maketitle
+
+\abstract{In this reflective report, I'll be reviewing my learning while reflecting on my projects and approaches for the first semester.}
+
+\section{Introduction}
+
+The three weakness I explore in this report are: time management, critical evaluation and communication. All of these skills are significant to me for professional practise and have been emphasised during my first semester.
+
+\section{Time Management}
+
+\subsection{Description - What happened? What is being examined?}
+
+One of my weaknesses was time management. With most tasks, it is crucial to follow a strict and thoughtful plan which accounts for all aspects of the project. For me, the relevance of this issue was for not anticipating all aspects of each project.
+
+\subsection{How has this impacted my learning and projects?}
+
+I found that my allocated work time had been misplaced. This not only lead to a low quality of work but also low quantity.
+
+\subsection{Interpretation - What is relevant about this idea?}
+
+Initially, I noticed that I did not feel confident in my abilities. At the time, I did not know how to overcome this issue which lead to frustration.
+
+\subsection{Outcome- What have I learned and what can be done in the future?}
+
+I now realise my time should have been more focused. For the future, I aim to make structured session objectives and expected completion times. If objectives are incomplete, then I must approach my lecturer or colleagues with my concerns. Next month, I shall examine my expectations of each week and whether my objectives were completed or guidance was sought. This focus would greatly contribute to my future career as a game developer.
+
+\section{Critical Evaluation}
+
+\subsection{Description - What happened? What is being examined?}
+
+Another of my weaknesses was critical evaluation. The ability to critique my own work allows for issues to be discussed and improvements to be made.
+
+\subsection{How has this impacted my learning and projects?}
+
+Unlike my previous education, these tasks required my own objective evaluation of work quality. This lead to being overly critical of my work, in a way that wasn't constructive.
+
+\subsection{Interpretation - What is relevant about this idea?}
+
+For me, the issue arose from low confidence. Subsequently, I felt inappropriately equipped to offer critical advice. In addition to this, I felt unfamiliar with marking others work with regards to coding practise.
+
+\subsection{Outcome- What have I learned and what can be done in the future?}
+
+I now realise that making reference to project objectives focuses evaluation. In addition, using objectives as a checklist proves each topic has been discussed. For me, in a career as a game developer, this skill would provide focused group and individual reflection. Next month, I shall examine my peer reviews as to whether I covered, and suggested improvements, for all points in my checklist.
```

Comment:
Adopting a structured approach is a good suggestion, as it can be measured and adapted.